src/test/java/com/capitalone/dashboard/collector/AuditCollectorTaskTest.java

```diff
@@ -2,14 +2,7 @@
 
 import com.capitalone.dashboard.collector.config.FongoConfig;
 import com.capitalone.dashboard.collector.config.TestConfig;
-import com.capitalone.dashboard.model.Dashboard;
-import com.capitalone.dashboard.model.AuditResult;
-import com.capitalone.dashboard.model.AuditType;
-import com.capitalone.dashboard.model.Audit;
-import com.capitalone.dashboard.model.AuditStatus;
-import com.capitalone.dashboard.model.DataStatus;
-import com.capitalone.dashboard.model.DashboardType;
-import com.capitalone.dashboard.model.CollectorItem;
+import com.capitalone.dashboard.model.*;
```

Comment:
@benj58xu can import only needed objects?

Reply:
Wonder if there is an intellij setting? It does this automatically.

Reply:
And I am also curious why people here insist a different style from others?

Reply:
Changed.